mne/viz/topo.py

```diff
@@ -426,6 +417,15 @@ def _rm_cursor(event):
     ax.xaxis.label.set_color(hvline_color)
     ax.yaxis.label.set_color(hvline_color)
 
+    if x_label is not None:
+        ax.set_xlabel(x_label)
```

Comment:
this needs to be done *after* `_setup_ax_spines` on line 410, which sets xlabel as `'Time (s)'`.  ylabel is not affected, but I thought it made sense to keep the `set_xlabel` and `set_ylabel` code together.